tools/buildroot_checksum.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/sh
+set -e
+write=0
+latest_commit_branch=""
+while getopts "f:wl:h" opt; do
+    case $opt in
+        f)
+            file=$OPTARG
+            ;;
+        # write to "checksum" file
+        w)
+            write=1
+            ;;
+        l)
+            latest_commit_branch=$OPTARG
+            ;;
+        h)
+            echo "Usage: $0 -f <path/to/bootrr.mk> [-w] [-l <branch>]"
+            echo "  -f <path/to/bootrr.mk>  Path to bootrr.mk"
+            echo "  -w                      Write to checksum file"
+            echo "  -l <branch>             Use latest commit from branch"
```

Comment:
I would say `-b` would be more intuitive for choosing the branch name.

Reply:
Changed